### ISSUE TITLE
Import audio

### ISSUE THREAD

**Issue by Inexpress** [nilfoer/gwaripper#8]
I guess this is more of a feature/documentation request, but is there any simple way to manually add files to the database? This would be useful for e.g. Patreon audios

**Comment by Inexpress** [comment]
Thank you, very much appreciated.

**Comment by Inexpress** [comment]
I began testing the functionality tonight. It works great, but figured I would report one minor issue relates to filename length. I tried adding an audio with a long title containing various tags. The result was:
—All the tags and everything were preserved in the main audio title. No problems there.
—The "Audio Page Title" and "Local Filename" don't seem to linebreak correctly. They extend well past the borders of the box, and push the audio/website/delete/heart buttons beyond the bounds of the UI unless I zoom far out.

At first I thought that was because 1) the "audio page title" is usually a short soundgasm title, and 2) the filenames (when audio is added automatically) usually contain spaces allowing for breaking, but these are replaced with underscores in the manual flow. But I checked some audios downloaded automatically earlier, which have long soundgasm titles, and in those cases the long "Audio Page Title" breaks fine.

Not sure if this photo helps at all. On the left, the non-wrapping text from the audio I added manually. On the right, wrapping text from the auto-added audio I mentioned. Wish I could diagnose the issue further but my web skills aren't up to scratch.
![image](https://user-images.githubusercontent.com/111848151/208680928-fbdfeceb-7107-4a32-b71e-23538878c2f7.png)
